test/callback.jl

```diff
@@ -304,6 +304,40 @@ using Base.Test
         @test solve(mod, suppress_warnings=true) == :UserLimit
     end
 
+    @testset "Callback exit on return StopTheSolver with $solver" for infosolver in info_solvers
```

Comment:
should be ``"Callback exit on return StopTheSolver with $infosolver"``

Reply:
Both are corrected now. Sorry I was not very experienced with @test structure.

Reply:
No apologies necessary, the JuMP test currently a bit difficult to work with.